### ISSUE TITLE
Build DLA engine failed

### ISSUE THREAD

**Issue by KexianShen** [NVIDIA/TensorRT#2307]
## Description

Building a DLA engine failed on orin, which returns:
```bash
[01/01/1970-02:48:31] [E] Error[2]: [nvmediaTensorCache.cpp::registerTensor::80] Error Code 2: Internal Error (Assertion nbTensors <= capacity() failed. Found a case where the cache is too small.)
[01/01/1970-02:48:31] [E] Error[2]: [builder.cpp::buildSerializedNetwork::607] Error Code 2: Internal Error (Assertion enginePtr != nullptr failed. )
[01/01/1970-02:48:31] [E] Engine could not be created from network
```

## Environment

**TensorRT Version**: 8.3.0.10
**NVIDIA GPU**: orin
**NVIDIA Driver Version**: orin linux sdk 6.0.2.0
**CUDA Version**: 11.4.14
**CUDNN Version**: 8.2.6.28

## Relevant Files

[onnx and calibration table](https://drive.google.com/drive/folders/1ZSLBjTSECw8buVtJdfglJsLPK2ZqxF2m?usp=sharing)

## Steps To Reproduce

- download files and scp files to orin
- run following command on orin
  ```bash
  trtexec --onnx=backboneV2.onnx \
  --saveEngine=model_bn.engine \
  --exportProfile=model_bn.json \
  --int8 \
  --calib=CalibrationTable \
  --useDLACore=0 \
  --allowGPUFallback \
  --iterations=100 \
  --sleepTime=200 \
  --idleTime=100 \
  --avgRuns=100 \
  --separateProfileRun \
  > model_bn.log
  ```

**Comment by zerollzeng** [comment]
Thanks, I've filed internal bug 3790543 to track this.

**Comment by KexianShen** [comment]
If you guys could build on orin with different environment, please let me know. Thanks.

**Comment by zerollzeng** [comment]
This issue has been fixed in The  8.5 release. which should come with the 6.0.5.0 or 6.0.6.0 SDK. Thanks

**Comment by wywywy01** [comment]
Is it fixed in lasted 8.5.1 release?

**Comment by zerollzeng** [comment]
Yes, will also be in the 8.5.1 release
